UI/window-basic-main-outputs.cpp

```diff
@@ -2011,22 +2011,23 @@ bool AdvancedOutput::ReplayBufferActive() const
 
 /* ------------------------------------------------------------------------ */
 
-bool BasicOutputHandler::SetupAutoRemux(const char *&ext)
+void BasicOutputHandler::SetupAutoRemux(const char *&ext)
```

Comment:
This seems out of place. Could you explain?

Reply:
The reason because it is now void is because the bool isn't needed with the refactored code.